src/amo/pages/UserProfile/index.js

```diff
@@ -173,7 +196,7 @@ export class UserProfileBase extends React.Component<InternalProps> {
 
     invariant(currentUser, 'currentUser is required');
 
-    if (currentUser.id === params.userId) {
+    if (`${currentUser.id}` === params.userId) {
```

Comment:
The test case was not realistic. `params.userId` is a string and `===` checks the type too 🤒

Reply:
I wasn't aware of this notation. I will use it!